### DISCUSSION TITLE
Ray Laravel Heroku Dynos Debug without ssh

### DISCUSSION THREAD

**Discussion by giacomeli** [spatie/laravel-ray#244] [Q&A]
Hi, i have some Laravel Apps running under Heroku dynos, that don't have access via ssh.  Can i connect those apps to Ray from another way?

**Comment by wit3** [comment]
Anyone has a suggestion about this? ray isn't compatible with serverless heroku? ( w/o ssh)

**Comment by tomascordero** [comment]
Any update this? I'd very much like to use Ray but all my apps run on heroku.
